changelog_unreleased/cli/12800.md

```diff
@@ -0,0 +1,24 @@
+#### [HIGILIGHT]Add `--cache` and `--cache-strategy` CLI option (#12800 by @sosukesuzuki)
+
+Two new CLI options have been added for a caching system similar to [ESLint's one](https://eslint.org/docs/user-guide/command-line-interface#caching).
+
+##### `--cache`
```

Comment:
Should we mention the cache location in this section, in case someone needs to clear it? You know, caching being one of the hardest things in computer science and all 😄

Reply:
[c463201](https://github.com/prettier/prettier/pull/12800/commits/c4632015708161c67675ba5d5810860b760f5ee5)